### DISCUSSION TITLE
Gemini的safety settings

### DISCUSSION THREAD

**Discussion by Koto0701** [plateaukao/einkbro#377] [Q&A]
感谢您开发了一部绝佳的软件，这是墨水屏上必不可少的APP！
请问要怎样才能调低Gemini的安全设置呢？它实在是太敏感了，才出现了一个cuckold就不给我翻译这段话了。

**Comment by plateaukao** [comment]
已經在程式裡有設定為最不敏感的狀態了，但沒用。Google 太小心謹慎了。
<img width="1066" alt="image" src="https://github.com/plateaukao/einkbro/assets/4084738/c5c79acf-b9f4-404b-babe-8f8a65d53a9a">

**Reply by Koto0701** [reply]
抱歉，可不可以直接关闭（设定成BLOCK_NONE），或者让用户自行调节安全设置呢？
我在您开发的另一个项目AskGPT中，把文件gemini_query.lua里的BLOCK_ONLY_HIGH全部替换成BLOCK_NONE后就没有再出现一有敏感词就不回答这种情况了。
这是谷歌对安全设置的说明https://ai.google.dev/gemini-api/docs/safety-settings?hl=zh-cn#python

**Reply by plateaukao** [reply]
之前文件上說無效的，看來後來有修正。我更新了 commit: https://github.com/plateaukao/einkbro/commit/fd08efa879be90934b77445a4b931ccc1e84241a

謝謝回報，我再去也修改一下 askgpt 的程式碼。

**Reply by Koto0701** [reply]
好的，谢谢！现在好用多了
